data/type_conversions.go

```diff
@@ -250,6 +250,17 @@ func ToBlob(v Value) ([]byte, error) {
 		return base64.StdEncoding.DecodeString(val)
 	case TypeBlob:
 		return v.asBlob()
+	case TypeArray:
+		a, _ := v.asArray()
+		b := make([]byte, len(a))
+		for i, e := range a {
+			v, err := e.asInt()
+			if err != nil || !(0 <= v && v <= 255) {
+				return nil, fmt.Errorf("cannot convert %T to Blob value", v)
```

Comment:
If the target array include a non-integer type value, like `data.Array{data.Float(0.0), data.Float(0.1)}`, then the error message is "cannot convert int64 to Blob value" because `e.asInt()` returns 0 when `e` is float, and the message is not correct.